magicgui/widgets/_function_gui.py

```diff
@@ -105,7 +105,7 @@ def __init__(
         if tooltips:
             _inject_tooltips_from_docstrings(function.__doc__, param_options)
 
-        self._function = function
+        self.__wrapped__ = function
```

Comment:
changing this to `__wrapped__` for potential future compatibility with https://docs.python.org/3/library/functools.html